---
layout: layouts/case.njk
title: Royal Society Conference on Chromosome Dynamics
summary: Secured financial support for the first international conference on computational models of Chromosome Dynamics.
date: 2025-09-08
tags: [case]
permalink: /work/grant-writing/
---

![programme](/images/programme.png)


## Goal 
- Define the current state of knowledge and identify challenges of future progress
- Present our research on the 3-dimensional distribution of chromatin states, itself a project driven both by experimental data and computational methods
- Make introductions between researchers with the aim to further the field of computational models in chromatin structure
- Define areas of possible future collaborations


### What we did
- Invited 20 Scientists of International standing to participate 
- Brought together leaders in their fields and introduced them to ongoing questions across domains
- Hosted and organised the meeting on the grounds of Chichely Hall



{% figure "Participants (L to R): Vivek Thacker, Eileen Nugent, Jonathan	Baxter, Peter Fraser, Paul Wiggins, Nicholas Kent, Rohit Pappu, Kazuhiro Maeshima, Megan King, Christian Haering, Philipp Korber, Martin	Howard, Steven Andrews, Karen Lipkow, Paul A. Bates, Mario	Nicodemi, Kerstin Bystricky, Leonie	Ringrose, Patrick Varga-Weisz" %}
![participants](/images/particpants.jpg)
{% endfigure %}

### Outcomes

The primary outcome of this meeting was to introduce the participants to researchers from otherwise fairly separate fields (experimentalists and compuational modellers). We also want to re-invigorate the field by highlighting the importance of modelling chromatin dynamics. The direct outcomes depend largely on the interaction of the participants and will only be apparent over time.
	


### Rationale & Motivation
The structure of chromatin has been notoriously difficult to determine. Composed of megabases of DNA and more than 200 known different proteins, our understanding of the exact dimensions, composition, biophysics and dynamics have changed dramatically over the years. Now that we are able to determine the structure of whole genomes, and combine this information with the increasing wealth of genome-wide epigenetic and gene expression data, we are beginning to see how the 3D structure of the entire genome has implications for regulation of gene expression and cellular development. Similarily, we are beginning to see how genome structure is being shaped by cellular events. In essence, gene regulatory events are accompanied, or even brought about by, changes in genome architecture.
This meeting will bring together computational and experimental researchers at the forefront of the chromatin field, with the aim to understand where the current boundaries of our understanding exactly lie. Computational models of chromatin need to be able to incorporate the latest data and be informed of current experimental approaches. And experimentists are eager to understand to what degree precisely models can help their research, experimentally and conceptually. Bringing these two approaches together is currently not done in other meetings, which is our motivation to host this conference. 
Being able to understand chromatin dynamics quantitatively will impact our understanding of gene regulation, cellular development and many disease phenotypes. 


We are confident that this meeting will be a catalyst for a number of collaborations for ourselves as well as for the participants. The problem at hand is a fundamental and large one which will require the combined effort of many research groups to make significant progress. 


### Justification
We investigate cellular and nuclear architecture using computation and experiments. We therefore have a precise view on the challanges faced by both fields. We have recently made significant advances and see great possibilities in computationally modelling the dynamics of genome structures. This is a strategically important part of our research, and can only be achieved by diving deep into the research community and by highlighting areas of communal interest. None of the current software tools is able to provide the level of detail and flexibility that we need when modelling the dynamic structure of the genome, and its interaction with diffusing and otherwise mobile proteins. This conference will bring together experts that understand different aspects of chromatin dynamics, inspiring the development of software that will enable us to reach our scientific goals. Chairing the conference will also help to establish myself in the chromatin field, in which I am not yet as well known as in the field of chemotaxis. 


Currently, the question of how genes are regulated runs into the ‘chicken and egg’ problem: transcription factors find their cognate DNA sequence within the context of chromatin - which dependends on other genes being expressed, which, in turn need to be regulated, ad infinitum. By incorporating genome structure, we can define a precise state that is causal for subsequent regulatory events. We now need to assess how to incorporate structural aspects into models of gene regulation and how to further develop computational approaches. 

### Benefits to UK science
As can be seen from the speaker list, many researchers in this area are located outside of the UK. The Babraham Institute nevertheless is at the forefront of determining the 3D organisation of whole genomes (c.f. Nagano et al, Nature 2013). We want to strengthen this overall leadership by adding the computational modelling component, which so far is represented less strongly in the UK. Providing clear intellectual and scientific leadership, we aim to influence the direction and speed with which the field moves, and create opportunities for future developments. 